CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bugfixes
 
+* Rename indexer metrics related to get operations. [#853](https://github.com/elastic/package-registry/pull/853)
+
```

Comment:
Should it be moved to another section in the changelog ?

Reply:
I think it can fit as a bugfix :slightly_smiling_face: Though it could be also seen as a breaking change.

Reply:
A bugfix is fine :)